startups/models.py

```diff
@@ -189,6 +183,35 @@ class Startup(Company):
         verbose_name="Development Stage",
         help_text="Current development stage of the startup"
     )
+    industry = models.ForeignKey(
+        Industry,
+        on_delete=models.PROTECT,
+        related_name="startups",
+        verbose_name="Industry",
+        help_text="Industry in which the startup operates"
+    )
+    funding_needed = models.DecimalField(
+        max_digits=12,
+        decimal_places=2,
+        default=0,
+        verbose_name="Funding Needed",
+        help_text="Amount of funding required by the startup"
+    )
+    team_size = models.PositiveIntegerField(
+        default=0,
+        validators=[MinValueValidator(1)],
+        verbose_name="Team Size",
+        help_text="Number of team members in the startup"
+    )
```

Comment:
The team_size field defaults to 0 but has a validator requiring at least 1. Recommend setting default=1 to match the validator.
```
team_size = models.PositiveIntegerField(
    default=1,
    validators=[MinValueValidator(1)],
    verbose_name="Team Size",
    help_text="Number of team members in the startup"
)
```